tests/journal/valuation.test

```diff
@@ -547,7 +547,7 @@ Budget performance in 2000Q1, valued at cost:
 ---++------------------------------------------------------------------------------------------------
    || 6 B [300% of 2 B]  7 B [350% of 2 B]  8 B [400% of 2 B]  21 B [350% of 6 B]  7 B [350% of 2 B] 
 
-# 48. budget report, valued at period ends.
+# 48. budget report, showing changes in period-end values.
 $ hledger -f- bal -MTA --budget --value=e
 Budget performance in 2000Q1, valued at period ends:
```

Comment:
I can add this as a fixup, but need a sanity check: the budget report will also show value changes, and we can title it "Budget performance in 2000Q1, period-end value changes:"  ?

Reply:
Oh I see, BudgetReport is separate and has not changed. 

Looks good to me, thank you!